sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -41,6 +44,14 @@ import scala.collection.JavaConverters._
 private[spark] object CosmosConfigNames {
   val AccountEndpoint = "spark.cosmos.accountEndpoint"
   val AccountKey = "spark.cosmos.accountKey"
+  val AuthType = "spark.cosmos.auth.type"
+  val SubscriptionId = "spark.cosmos.auth.aad.subscriptionId"
```

Comment:
I doubt we need subscriptionid and resourcegroupname - it might make it easier to address the account resource - but has nothing to do with auth - and there should be a way to find the account name without already knowing subid and rg I believe.

Reply:
Discussed offline, since resourceGroupName is a required parameters in ManagementSDK, so will expose it. Will follow up offline with Management SDK team